libnavigation-core/src/main/java/com/mapbox/navigation/core/reroute/MapboxRerouteController.kt

```diff
@@ -154,7 +156,8 @@ internal class MapboxRerouteController @VisibleForTesting constructor(
                         val modifiedRerouteOption = compositeRerouteOptionsAdapter.onRouteOptions(
                             routeOptionsResult.routeOptions
                         )
-                        request(callback, modifiedRerouteOption)
+                        //TODO: can it be null?
```

Comment:
No, otherwise it wouldn't be `RouteOptionsUpdater.RouteOptionsResult.Success`. But let's maybe store `tripSessino.getRouteProgress` in a local variable to fix it once and for all in this method.

Reply:
done